frontend/src/metabase/query_builder/components/NativeQueryEditor/AceEditor.global.css

```diff
@@ -0,0 +1,110 @@
+:local(.editor) {
+  height: 100%;
+  width: 100%;
+  flex-shrink: 1;
+  flex-grow: 1;
+
+  &.ace_editor {
+    background-color: var(--mb-color-bg-light);
+    color: var(--mb-color-text-dark);
+  }
+
+  & .ace_search {
```

Comment:
These `&` prefixes should not be necessary, but for some reason they don't get compiled away by `postcss-nesting`.

I've set up a [PR](https://github.com/metabase/metabase/pull/50437) to fix that.